executor/executor_test.go

```diff
@@ -2202,7 +2202,7 @@ func (s *testContextOptionSuite) TestCoprocessorPriority(c *C) {
 	tk.MustQuery("select count(*) from t")
 	tk.MustExec("update t set id = 3")
 	tk.MustExec("delete from t")
-	tk.MustExec("insert into t values (2)")
+	tk.MustExec("insert into t select * from t limit 2")
```

Comment:
what is the priority of "insert into t select * from t"?

Reply:
Normal